icons/symbolic/scalable/apps/facebook-symbolic.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" class="facebook"><path d="M9.239,16V8.7h2.45l.367-2.844H9.239V4.041c0-.824.229-1.385,1.41-1.385h1.506V.112A20.155,20.155,0,0,0,9.96,0,3.427,3.427,0,0,0,6.3,3.76v2.1H3.845V8.7H6.3V16Z"/></svg>
```

Comment:
Is the `class="facebook"` necessary for theming? If not, then we should remove it. (Same for Twitter and Whatsapp)

Reply:
yes, because this is embedded in the html articles and the class is used on the theme